kbs_protocol/src/lib.rs

```diff
@@ -113,7 +116,8 @@ impl KbsProtocolWrapper {
         match attest_response.status() {
             reqwest::StatusCode::OK => {
                 self.authenticated = true;
-                Ok(())
+                let token = attest_response.json::<serde_json::Value>().await?["token"].to_string();
```

Comment:
Would make sense to check here the token is really base64 encoded?

Reply:
the token is supposed to be a json-web-token which has base64 components but is simply a base64 encoded string.